icu4c/source/i18n/measunit_extra.cpp

```diff
@@ -321,6 +321,12 @@ class Token {
 
 class Parser {
 public:
+    /**
+     * Factory function for parsing the given identifier.
+     *
+     * @param source The identifier to parse. This function does not make a copy
+     * of source: the data source points at must outlive the parser.
```

Comment:
Check the grammar of this sentence

Reply:
Rewrote it, thanks! ("data" was way too ambiguous, looking like an adjective rather than a noun.)